src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Tooltip from '../Tooltip';
+import Link from '../';
+
+
+describe('Tooltip Component Suit', () => {
+    describe('Test children property', () => {
```

Comment:
Add cases of property for following:
1. Not set 
2. Set valid values
3. Set undefined/false value.